gobblin-service/src/main/java/org/apache/gobblin/service/modules/spec/JobExecutionPlan.java

```diff
@@ -54,11 +57,11 @@
 
   public static class Factory {
     public static final String JOB_NAME_COMPONENT_SEPARATION_CHAR = "_";
-    private static final Random random = new Random();
+    public static final String METRICS_REPORTING_CONFIG_PREFIX = "metrics.reporting";
```

Comment:
Is this the scope name? Can we have a more scoped name like "gobblin.service.metrics.reporting"?